src/Sarif.Viewer.VisualStudio/Tags/ISarifLocationTaggerService.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+namespace Microsoft.Sarif.Viewer.Tags
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    [Guid("774C62D3-203A-493C-8801-BEA59FE46CA1")]
+    internal interface ISarifLocationTaggerService
+    {
+        /// <summary>
+        /// Causes a tags changed notification to be sent out from all known taggers.
+        /// </summary>
+        /// <remarks>
+        /// The primary use of this is to send a tags changed notification when a "text view" is already open and visible
+        /// and a tagger is active for that "text view" and a SARIF log is loaded via an API.
+        /// </remarks>
+        void RefreshAllTags();
+
+        /// <summary>
+        /// Called when a new tagger is created which allows <see cref="RefreshAllTags"/> to call into the tagger.
```

Comment:
>Called when a new tagger is created which allows <see cref="RefreshAllTags"/> to call into the tagger. [](start = 12, length = 102)

Adds a new tagger to the list of taggers so that "RefreshAllTags" can notify the tagger to refresh its tags.